server/app/models/grid_service.rb

```diff
@@ -147,7 +147,7 @@ def stack_exposed?
   end
 
   def deploy_pending?
```

Comment:
In #2221, I just called this `deploying?`, and renamed the existing `deploying?` method to `deploy_running?`.